cli/src/etos_client/lib/test_result_handler.py

```diff
@@ -177,7 +177,7 @@ def collect_sub_suites(self, main_test_suite_id):
                 continue
             finished = self.finished(event_id)
             if finished is not None:
-                self.test_suite_started[event_id]["finished"] = finished is not None
+                self.test_suite_started[event_id]["finished"] = True
```

Comment:
This is no a fix to the actual problem. The previous way of doing it bugged me since we already do that check on line 179.